src/main/java/com/spectralogic/ds3cli/views/cli/HeadObjectView.java

```diff
@@ -0,0 +1,56 @@
+package com.spectralogic.ds3cli.views.cli;
+
+import com.bethecoder.ascii_table.ASCIITable;
+import com.bethecoder.ascii_table.ASCIITableHeader;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.common.base.Joiner;
+import com.spectralogic.ds3cli.View;
+import com.spectralogic.ds3cli.models.HeadObjectResult;
+import com.spectralogic.ds3client.networking.Metadata;
+
+import java.util.List;
+import java.util.Set;
+
+public class HeadObjectView implements View<HeadObjectResult> {
+    @Override
+    public String render(final HeadObjectResult obj) throws JsonProcessingException {
```

Comment:
Does this really throw a JsonProcessingException?  I don't see any JSON'ing happening..?